client/Components/Decks/DeckRow.jsx

```diff
@@ -1,10 +1,8 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 import moment from 'moment';
-
```

Comment:
I like to have a separation of 'outside' dependencies, vs 'inside' dependencies which is why this blanks space exists :)

Reply:
fixed in https://github.com/keyteki/keyteki/pull/488/commits/856c00d0295cf9012fe84bafd8f3ccd160a120df